zeppelin-web-angular/src/app/pages/workspace/notebook/paragraph/code-editor/code-editor.component.ts

```diff
@@ -61,6 +61,7 @@ export class NotebookParagraphCodeEditorComponent implements OnChanges, OnDestro
   private monacoDisposables: IDisposable[] = [];
   height = 18;
   interpreterName?: string;
+  editorSettingTriggerAllowed: boolean = false;
```

Comment:
Could we perhaps add a comment explaining  why this property is introduced here?